common/test/keyboards/web_context_tests/README.md

```diff
@@ -0,0 +1,37 @@
+web_context_tests keyboard
+==============
+
+Version 1.0
+
+Description
+-----------
+web_context_tests is used for testing selection rules. It has a single rule:
+
+```
+  'abc' + 'd' > '!'
```

Comment:
Is this updated to include the SMP rules tested in other PRs?

One minor nit as that old Android devices (API 21) could use a keyboard font 😢

Reply:
ah, I found that keyboard on #6272